src/style.css

```
@@ -1 +1,80 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --link-color: #00acdc;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding: 0 50px;
+}
+
+.header__container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1100px;
+  margin: 0 auto;
+}
+
+.logo__img {
+  vertical-align: top;
+}
+
+.menu__list {
+  display: flex;
+  list-style: none;
+}
+
+.menu__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.menu__link {
+  display: inline-block;
+  padding: 23px 0;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+}
+
+.menu__link:hover {
+  color: var(--link-color);
+}
+
+.is-active {
+  color: var(--link-color);
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  width: 37px;
```

do not hardcode this, it should fit any link here
```suggestion
  width: 100%;
```

---

Right! Next time I'll be more thoughtful. Thank's!